cardano-db-sync/src/Cardano/DbSync/Cache.hs

```diff
@@ -144,6 +157,53 @@ getCacheStatistics cs =
     UninitiatedCache -> pure initCacheStatistics
     Cache ci -> readTVarIO (cStats ci)
 
+readCacheEpoch :: Cache -> IO (Maybe CacheEpoch)
+readCacheEpoch cache =
+  case cache of
+    UninitiatedCache -> pure Nothing
+    Cache ci -> readTVarIO (cEpoch ci)
+
+writeCacheEpoch ::
+  MonadIO m =>
+  Cache ->
+  CacheEpoch ->
+  ReaderT SqlBackend m ()
+writeCacheEpoch cache cacheEpoch =
+  case cache of
+    UninitiatedCache -> pure ()
+    Cache ci -> liftIO $ atomically $ writeTVar (cEpoch ci) $ Just cacheEpoch
+
+-- write a block into the cache
+writeBlockToCacheEpoch ::
```

Comment:
I'm assuming this won't be used eventually

Reply:
yeah think now I write everything in one go to the cache not the block on it's own. Kept it around just incase but I'll make a note to delete it if I don't.